dist-build/android-build.sh

```diff
@@ -1,11 +1,11 @@
 #! /bin/sh
 
 if [ -z "$NDK_PLATFORM" ]; then
-  export NDK_PLATFORM="android-19"
+  export NDK_PLATFORM="android-16"
```

Comment:
Why revert to `android-16`? Looks like the lowest version supported by the NDK is now `19`.

Reply:
I didn't notice this. I think it was 16 in the stable version.